backend/core/src/listings/entities/listing.entity.ts

```diff
@@ -358,6 +359,19 @@ class Listing extends BaseEntity {
       this.waitlistCurrentSize < this.waitlistMaxSize
     )
   }
+
+  @ManyToOne(() => ReservedCommunityType, { eager: true, nullable: true })
+  @JoinColumn()
```

Comment:
I don't think @JoinColumn() is necessary here and we may want to update the listing service to include a leftJoinAndSelect for reservedCommunityType.